EssentialFeed/EssentialFeed/Image Comments/Image Comments API/RemoteImageCommentsLoader.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  RemoteImageCommentsLoader.swift
+//  EssentialFeed
+//
+//  Created by Lukas Bahrle Santana on 20/01/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+
+public final class RemoteImageCommentsLoader: ImageCommentsLoader {
```

Comment:
Too many line breaks. Should be just one.

```suggestion
import Foundation

public final class RemoteImageCommentsLoader: ImageCommentsLoader {
```

Please, make sure the code formatting is consistent everywhere - consistency is important as it makes code easier to follow and understand.